package.json

```diff
@@ -74,6 +74,7 @@
   "dependencies": {
     "@fortawesome/fontawesome-svg-core": "^6.5.2",
     "@fortawesome/free-solid-svg-icons": "^6.5.2",
-    "@fortawesome/react-fontawesome": "latest"
+    "@fortawesome/react-fontawesome": "latest",
+    "arcgis-pbf-parser": "^0.0.4"
```

Comment:
How is this being used? Is this a leftover from earlier implementation?

Reply:
Woops, yea that's a leftover.